TraceScripts/Android/PlayFabMultiplayerLogger.json

```diff
@@ -0,0 +1,14 @@
+{
+  "enabled": true,
+  "bufferSize": 262144,
+  "maxNumberOfItemsInBatch": 500,
+  "maxBatchWaitTimeInSeconds": 1,
+  "readBufferWaitTimeInMilliseconds": 1,
+  "logFolder": "/sdcard/PlayFabLogs/log/",
+  "logLevel": "INFO",
```

Comment:
>INFO [](http://example.com/codeflow?start=15&length=4)

Make VERBOSE as default, INFO logs are not very useful

Reply:
I had previously suggested Nassos use INFO in the MacOS and Apple json files since I had assumed INFO was the more reasonable default. If that suggestion was incorrect we should probably fix those as well

Reply:
Agreed, I did some testing last week and VERBOSE logs were a great deal more useful. @SahilAshar could you make these changes also?

Reply:
Yup, just had the same convo offline with Push. Will update them all accordingly.